src/librustc_typeck/check/coercion.rs

```diff
@@ -123,10 +130,13 @@ fn success<'tcx>(adj: Vec<Adjustment<'tcx>>,
 }
 
 impl<'f, 'gcx, 'tcx> Coerce<'f, 'gcx, 'tcx> {
-    fn new(fcx: &'f FnCtxt<'f, 'gcx, 'tcx>, cause: ObligationCause<'tcx>) -> Self {
+    fn new(fcx: &'f FnCtxt<'f, 'gcx, 'tcx>,
```

Comment:
Nit: these days, I tend to move over to rustfmt style when changing a sig anyway

```
fn new(
    fcx: ...
    cause: ...
    allow_two_phase: ...,
) -> Self {
```